tests/roles/mariadb_copy/templates/pre_checks.bash

```diff
@@ -5,7 +5,7 @@
 {{ mariadb_copy_shell_vars_dst }}
 
 # Test connection to podified DBs (show databases)
-oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
+oc run mariadb-client ${MARIADB_CLIENT_ANNOTATIONS} --image $MARIADB_IMAGE -i --rm --restart=Never -- \
```

Comment:
This change needs to be done in docs too.

Reply:
ok will add those to my pr